webapi/tct-colors-css3-tests/suite.json

```diff
@@ -26,6 +30,15 @@
                     "PACK-TOOL-ROOT/resources/testharness": "resources",
                     "PACK-TOOL-ROOT/resources/webrunner": "webrunner"
                 }
+            },
+            "subapp-list": {
+                "css3-color-app": {
+                    "apk-common-opts": "--enable-remote-debugging",
+                    "apk-icon-opt": "",
+                    "copylist": {
+                        "../colors/csswg/bdd": "bdd"
```

Comment:
please update "html_files" -> csswg, let's use the bdd parent folder name during copy into subapp